.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Also tell us, what did you expect to happen? If applicable, add screenshots to help explain your problem.
+      description: Tell us what you see!
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-did-you-expected
+    attributes:
+      label: What did you expected to happened
+      description: Tell us what you expected!
+      placeholder: 
```

Comment:
```suggestion
      label: What did you expect to happen?
      description: Tell us what you wanted to see!
      placeholder: Ideally, it would have made it all the way through installation
```